patroni/api.py

```diff
@@ -535,6 +535,18 @@ def do_GET_metrics(self) -> None:
         metrics.append("patroni_xlog_paused{0} {1}"
                        .format(scope_label, int(postgres.get('xlog', {}).get('paused', False) is True)))
 
+        if postgres.get('server_version', 0) > 90600:
```

Comment:
```suggestion
        if postgres.get('server_version', 0) >= 90600:
```

I still need to take a look at the PR, but at a first glance I guess this should be `>=` instead of `>` as `pg_stat_get_wal_receiver` is available in 9.6. I could see that in the place `pg_stat_get_wal_receiver` is actually called it has a `>=` already.